### DISCUSSION TITLE
Livewire Action not working in v2.5.2

### DISCUSSION THREAD

**Discussion by JeffGepiga** [livewire/livewire#3265] [7. Bugs (archived)] [answered]
I think there is an error in production mode of livewire, the script is not being properly rendered,
please see attached image for reference
![image](https://user-images.githubusercontent.com/51604875/124904398-8d095200-e017-11eb-9ee6-1ccb2a90a83e.png)

**Comment by joshhanley** [comment, marked as answer]
@theonly27 for some reason the fix applied in 2.5.1 got reverted by mistake. I have submitted a PR with the fix #3263

Hope this helps!

**Reply by JeffGepiga** [reply]
Thanks!
